scroll_derive/src/lib.rs

```diff
@@ -128,6 +128,14 @@ fn impl_pwrite_field(ident: &proc_macro2::TokenStream, ty: &syn::Type) -> proc_m
             _ => panic!("Pwrite derive with bad array constexpr"),
         },
         syn::Type::Group(group) => impl_pwrite_field(ident, &group.elem),
+        syn::Type::Reference(reference) => match *reference.elem {
+            syn::Type::Slice(_) => quote! {
+                dst.gwrite_with(self.#ident, offset, ())?
```

Comment:
why is a `()` ctx passed here?

Reply:
refer https://github.com/m4b/scroll/blob/3cffc5a2a1eaf4c7d698f3ffba2509523ece1875/src/ctx.rs#L713-L716

Reply:
:sweat_smile: haha yes i forgot `pwrite` for `&[u8]` has no ctx, though that is a bit quirky/assymetric, as it's `pread` has a usize ctx (e.g., how many things to read). r